server/weaverbird/steps/lowercase.py

```diff
@@ -0,0 +1,18 @@
+from pandas import DataFrame
+from pydantic import Field
+
+from weaverbird.steps import BaseStep
+from weaverbird.types import ColumnName, DomainRetriever, PipelineExecutor
+
+
+class LowercaseStep(BaseStep):
```

Comment:
just to be consistent with typescript steps:
```suggestion
class ToLowerStep(BaseStep):
```